### ISSUE TITLE
undefined: imap.NewSeqSet

### ISSUE THREAD

**Issue by matiux** [emersion/go-imap-move#3]
Hi,
after the fix for this bug `https://github.com/emersion/go-imap/issues/127` I get this error:

```
# gitlab.com/acme/project/vendor/github.com/emersion/go-imap-move
vendor/github.com/emersion/go-imap-move/command.go:35: undefined: imap.NewSeqSet
```

Is related?

P.s. I use Glide to handle vendor

**Comment by emersion** [comment]
Sorry, this extension was out-of-date. Fixed in 297ce2310a5ef363c43bdf7e668ac86ee8e87444

**Comment by matiux** [comment]
Thanks :+1:
